### PR TITLE
Add WayPoints param in GetWayPoints response

### PR DESCRIPTION
**Pull request by OHerasym**
### PR Changes:

- Add WayPoints param in GetWayPoints response

Related: [APPLINK-25933](https://adc.luxoft.com/jira/browse/APPLINK-25933)

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @dtrunov , @anosach-luxoft , @LuxoftAKutsan , @TMelnyk

File: src/components/interfaces/MOBILE_API.xml [hunk 1]
```diff
@@ -5280,6 +5280,9 @@
 	<param name="info" type="String" maxlength="1000" mandatory="false" platform="documentation">
 	   <description>Provides additional human readable info regarding the result.</description>
 	</param>
+	<param name="wayPoints" type="LocationDetails" mandatory="false" array="true" minsize="1" maxsize="10">
+	   <description>See LocationDetails</description>
+	</param>
   </function>
```
Comment (dtrunov) [inline src/components/interfaces/MOBILE_API.xml]: @OHerasym should add this parameter to HMI_API.xml

Comment (OHerasym) [inline src/components/interfaces/MOBILE_API.xml]: @dtrunov , param in HMI_API already added

**Comment by LuxoftAKutsan** [comment]
Reviewed.

**Comment by TMelnyk** [comment]
reviewed

**Comment by dtrunov** [comment]
Reviewed

**Comment by anosach-luxoft** [comment]
Reviewed

**Pull request merged**